fastembed/image/onnx_embedding.py

```diff
@@ -29,9 +29,28 @@
         },
         "model_file": "model.onnx",
     },
+    {
+        "model": "AndrewOgn/vitb16-unicom",
+        "dim": 768,
+        "description": "Unicom vitb16_unicom from open-metric-learning",
+        "size_in_GB": 0.82,
+        "sources": {
+            "hf": "AndrewOgn/vitb16_unicom",
+        },
+        "model_file": "vitb16_unicom.onnx",
+    },
+    {
+        "model": "AndrewOgn/vitb32-unicom",
+        "dim": 512,
+        "description": "Unicom vitb32_unicom from open-metric-learning",
+        "size_in_GB": 0.48,
+        "sources": {
+            "hf": "AndrewOgn/vitb32_unicom",
+        },
+        "model_file": "vitb32_unicom.onnx",
```

Comment:
can we name the files model.onnx ? Looks to be some kind of a convention

Reply:
vit models usually named like this rather than vitb16

sentence-transformers/clip-ViT-B-32